package.json

```diff
@@ -133,6 +134,7 @@
     "stylus": "0.54.5",
     "superagent": "3.7.0",
     "superagent-bluebird-promise": "3.0.2",
+    "terser-webpack-plugin": "^1.3.0",
```

Comment:
Terser is the default minifier that webpack uses. You very likely don't need to include it as a dependency.

Reply:
Looks like we do indeed need this.